src/averageAges.js

```diff
@@ -37,7 +51,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenFormPeople
+    = people.filter(person => person.sex === 'f');
+
+  const womenWithChildren
+    = womenFormPeople
+      .filter(woman => people
+        .some((person) => person.mother === woman.name));
```

Comment:
can be done in one filter, just a filter with two conditions inside

Reply:
thanks fixed